paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentSheetViewModel.kt

```diff
@@ -432,6 +434,7 @@ internal class PaymentSheetViewModel @Inject internal constructor(
         checkout(PaymentSelection.GooglePay, CheckoutIdentifier.SheetTopWallet)
     }
 
+    @Suppress("LongMethod", "CyclomaticComplexMethod", "ComplexCondition")
```

Comment:
maybe a way to avoid making this method longer could be have `confirmPaymentSelection` handle the new cvc recollection logic, instead of adding it to `checkout`?